server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/ec.js

```diff
@@ -192,35 +242,12 @@ function refresh() {
  * Generates the compactions table
  */
 function refreshECTables() {
```

Comment:
I think the new table needs to be added to this method, I'm not seeing it refresh. Also, you can use `ajaxReloadTable()` from functions.js here.